src/parser.rs

```diff
@@ -77,16 +80,18 @@ pub fn extract_flash_algo(
     let algorithm_binary = crate::algorithm_binary::AlgorithmBinary::new(&elf, &buffer)?;
     algo.instructions = base64::encode(&algorithm_binary.blob());
 
+    let code_section_offset = algorithm_binary.code_section.start;
```

Comment:
Did you check if this works with the ARM algos? Or should I check this? :)

Reply:
I didn't check yet. With my self-written flash loader it works :wink: